### PR TITLE
Add note about repository size limitations

### PR DESCRIPTION
**Pull request by mikita-haurykau**

### CODE REVIEW

File: content/en/docs/refguide/general/system-requirements.md [hunk 1]
```diff
@@ -88,6 +88,20 @@ For active development and running your application locally, your app folder sho
...
 
+Git service providers have size limitations related to commits and repositories.
+
+Studio Pro does not support Large File Storage (LFS) for any provider.
+
+You can find a list of known limitations below, however, we advise you to check the most up-to-date limitations on respective web sites before considering using a certain service provider.
+
+| Vendor      | Repository Size limit                      | Push Limit               | Non-LFS file size limit | LFS file size limit |
+| ----------- | ------------------------------------------ | ------------------------ | ----------------------- | ------------------- |
+| GitHub      | Warnings on 5 GB, 100 GB theoretical limit | 2 GB (from archived doc) | 100 MB (warning 50 MB)   | 5 GB                |
```
Comment (MariaShaposhnikova) [inline content/en/docs/refguide/general/system-requirements.md]: @mikita-haurykau , what does "2 GB (from archived doc) " mean? Is information taken from an archived document if the GitHub website?

"Warnings on 5 GB" and "100 MB (warning 50 MB)" what warnings do we mean here?

Comment (mikita-haurykau) [reply content/en/docs/refguide/general/system-requirements.md]: >Is information taken from an archived document if the GitHub website?
yes
>"Warnings on 5 GB" and "100 MB (warning 50 MB)" what warnings do we mean here?
git warnings that show up in CLI output. StP doesn't show any warnigns like this

Comment (MariaShaposhnikova) [reply content/en/docs/refguide/general/system-requirements.md]: @mikita-haurykau , if the document was archived, I am not sure that this is best to put it on our website.

**Comment by MariaShaposhnikova** [comment]
Information in this PR needs to be copied to refguide9.

File: content/en/docs/refguide/general/system-requirements.md (hunk 1 above)
Comment (MariaShaposhnikova) [reply content/en/docs/refguide/general/system-requirements.md]: Discussed that I will change it to "information unavailable" instead.

**Comment by MariaShaposhnikova** [comment]
I've recreated and merged this PR in this one: https://github.com/mendix/docs/pull/5950, because a new major version was introduced and it was easier to reproduce it.

**Pull request closed without merging**